extensions.json

```
@@ -1506,7 +1506,7 @@
     },
     {
       "id": "rebornix.ruby",
-      "repository": "https://github.com/rubyide/vscode-ruby",
+      "download": "https://github.com/rubyide/vscode-ruby/releases/download/v0.28.0/ruby-0.28.0.vsix",
```

Forgot to undo this:

```suggestion
      "repository": "https://github.com/rubyide/vscode-ruby",
```